tests/flytekit/unit/core/test_array_node_map_task.py

```diff
@@ -464,3 +469,39 @@ def wf():
 
     with pytest.raises(AssertionError):
         wf.compile()
+
+
+def test_load_offloaded_literal(tmp_path, monkeypatch):
+    @task
+    def say_hello(name: str) -> str:
+        return f"hello {name}!"
+
+    ctx = context_manager.FlyteContextManager.current_context()
+    with context_manager.FlyteContextManager.with_context(
+            ctx.with_execution_state(
+                ctx.execution_state.with_params(mode=context_manager.ExecutionState.Mode.TASK_EXECUTION)
+            )
+    ) as ctx:
+        list_ints = ["a", "b", "c"]
```

Comment:
nit: can we rename this variable to `list_strs`?

Reply:
done